certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1630,6 +1630,1702 @@ public class Script : ScriptBase
       }
     }
 
+    // Composite Templates Request Body (please toggle up when not working on this part of code)
+    if (operationId.Equals("StaticResponseForCompositeTemplates", StringComparison.OrdinalIgnoreCase))
```

Comment:
Did you try reusing StaticResponseForAnchorTabSchema instead of duplicating whole bunch of code?

Reply:
Yes, I tried to use that. But that function has a different output structure that only includes a single tab type. Whereas this one includes much more.. Thus it looks like code duplication.